pkgs/by-name/li/libhttpserver/package.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, autoconf
+, automake
+, libtool
+, gnutls
+, libmicrohttpd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libhttpserver";
+  version = "0.19.0";
+
+  src = fetchFromGitHub {
+    owner = "etr";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Pc3Fvd8D4Ymp7dG9YgU58mDceOqNfhWE1JtnpVaNx/Y=";
+  };
+
+  nativeBuildInputs = [ autoconf automake libtool ];
+
+  buildInputs = [ gnutls libmicrohttpd ];
+
+  enableParallelBuilding = true;
+
+  postPatch = ''
+    patchShebangs ./bootstrap
+  '';
+
+  configurePhase = ''
+    runHook preConfigure
+    ./bootstrap
+    ./configure --prefix=$out --enable-same-directory-build
+    runHook postConfigure
+  '';
+
+  meta = with lib; {
+    description = "C++ library for creating an embedded Rest HTTP server (and more)";
+    homepage = "https://github.com/etr/libhttpserver";
+    license = licenses.lgpl21Plus;
+    maintainers = with maintainers; [ pongo1231 ];
+    platforms = platforms.unix;
```

Comment:
The build fails for x86_64-darwin: `configure: error: cannot find required auxiliary files: ltmain.sh`

Either set platforms to linux or mark it as broken for darwin.

Reply:
The library should run on mac from what I've gathered so I'll set it as broken on darwin for now - I won't be able to fix it for that platform however.

Reply:
Yeah that's alright. Marking it as broken indicates that it should work, but doesn't at the moment. I'd put a comment with the error message next to the broken = line, to give future readers some context.

Lots of maintainers can't test Darwin builds of their packages.